result_diamond.py

```diff
@@ -0,0 +1,37 @@
+import math
+
+
+class Rhombus:
+    def __init__(self, side_a, angle_a):
+        self.side_a = side_a  # Використовуємо __setattr__
+        self.angle_a = angle_a  # Використовуємо __setattr__
+
+    def __setattr__(self, name, value):
+        """Перевірки для атрибутів."""
+        if name == "side_a":
+            if value <= 0:
+                raise ValueError("Сторона ромба повинна бути більше 0.")
+        elif name == "angle_a":
+            if value <= 0 or value >= 180:
+                raise ValueError("Кут ромба повинен бути більше 0 і менше 180 градусів.")
+            # Автоматично обчислюємо кут β
+            object.__setattr__(self, "angle_b", 180 - value)
+
+        # Встановлюємо значення атрибута
+        object.__setattr__(self, name, value)
+
+    def perimeter(self):
+        """Обчислює периметр ромба."""
+        return 4 * self.side_a
+
+    def info(self):
```

Comment:
Instead of info you need to use `def __repr__`  or  `def __str__` then you will be able to do `print(romb)`